ts/store/actions/wallet/wallets.ts

```diff
@@ -10,6 +10,12 @@ import {
   WalletResponse
 } from "../../../types/pagopa";
 import { PayloadForAction } from "../../../types/utils";
+import { GlobalState } from "../../reducers/types";
+import { WalletsState } from "../../reducers/wallet/wallets";
+
+// Selectors
+export const walletsSelector = (state: GlobalState): WalletsState =>
```

Comment:
We should avoid to write a selector where actions are defined
Could you move it in `ts/store/reducers/wallet/wallets.ts`?

Reply:
Sure. Modified in [0f0f17a](https://github.com/pagopa/io-app/pull/2490/commits/0f0f17a07d34a3990fe3039b2f5e6eb7fc938893)